ai-platform-unified/notebooks/unofficial/model_monitoring/model_monitoring.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Instead of gs://mco-mm/churn, it should live in a DevRel bucket that is publicly accessible. Let me figure out exactly where.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/318/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I'm working on getting a project exception to overground but will use this in the meantime. Happy to move it to an official devrel bucket if you have one.